SETUP.md

```diff
@@ -1 +1,37 @@
+# Getting setup
 
+We will be using Anaconda to install and manage a Python 'environment' for us to work in. Rather than using any pre-installed Python 
+interpretors this has the advantage that it will not conflict with any existing libraries, and everyone will be using the same verisons.
+
+## 1. Install Anaconda
+
+If you haven't already got Anaconda installed, download and install it from: https://www.continuum.io/downloads. Choose Python 3.5 as this 
+is the latest version of Python.
+
+## 2. Create a new environment for the workshop
+
+Now you have installed Anaconda we can create an evironment to work in. Open your terminal (or 'Anaconda Prompt' on Windows) and type:
+
+``` $ conda create -c conda-forge -n python_workshop cis jupyter```
```

Comment:
I'm using extra dependencies for the barotropic model (pyspharm from conda-forge, geoviews from ioam). Pyspharm appears not to be available on win64... Not sure what to do about that.